README.md

```diff
@@ -47,6 +47,7 @@ Download this free e-book on [Porting existing ASP.NET apps to .NET Core](https:
 
 1. This tool uses MSBuild to work with project files. Make sure that a recent version of MSBuild is installed. An easy way to do this is to [install Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
 2. This tool requires that your project builds. This may include installing [install Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) to ensure build SDKs (such as for web applications, etc) are available.
+3. The tool may produce long file paths during the course of its execution, in order to not run into PathTooLongException either change the LongPathsEnabled setting under registry or execute upgrade-assistant from a folder location with a shorter path.
```

Comment:
The long path issues show up while developing the tool (building the solution, etc.) not while using it. So, we should include this with tips on contributing to the repo rather than in pre-reqs for using it.

Reply:
Moved to Contributing guide.